src/main/java/com/coveo/pushapiclient/CatalogSource.java

```diff
@@ -0,0 +1,17 @@
+package com.coveo.pushapiclient;
+
+public class CatalogSource {
+
+    public CatalogSource(String apiKey, String organizationId) {
```

Comment:
Should the constructor take an [initial url](https://github.com/coveo/push-api-client.java/pull/29/files) instead of an orgId?

Atm, the AdminUI directs a dev to use a url (which contains the org and source id), and an api key.

It would be nice for the sdk to follow this flow, or, if we disagree with it, to build the sdk, and adjust the AdminUI to make it easy to get the values needed by the constructor,

Reply:
I'm fine with that. that way, we don't need to expose another object for region and environment.